src/Assets/analytics.ts

```diff
@@ -0,0 +1,7 @@
+export default function dataLayerPush(obj: any) {
```

Comment:
What do you think about creating a function for each of the different param object types so that we don't have to declare the data type as `any` here?